nexus/completion.go

```diff
@@ -95,6 +126,12 @@ type OperationCompletionUnsuccessful struct {
 	Header http.Header
 	// State of the operation, should be failed or canceled.
 	State OperationState
+	// OperationID is the unique ID for this operation. Used when a completion callback is received before a started response.
+	OperationID string
+	// StartTime is the time the operation started. Used when a completion callback is received before a started response.
+	StartTime time.Time
+	// StartLinks are the links attached to the started response. Used when a completion callback is received before a started response.
+	StartLinks []Link
```

Comment:
I'm wondering if it's worth just calling these `Links`. Because there's no difference between a start link and a complete link.

Also, I'd improve the docstring here:

```suggestion
	// StartLinks are used to link back to the operation when a completion callback is received before a started response.
	StartLinks []Link
```

Reply:
I was thinking users may want to differentiate between links that should be attached to a fabricated start event and links that should be attached to the complete event. But maybe that is a Temporal-specific implementation detail?

Reply:
This should've been called `Links` IMHO. I will rename and submit it with other breaking changes I have planned.